misc/debug-adapter/modules/debug-adapter-core/src/main/java/org/ballerinalang/debugadapter/JBallerinaDebugServer.java

```diff
@@ -478,6 +499,15 @@ private Breakpoint toBreakpoint(SourceBreakpoint sourceBreakpoint, Source source
         return breakpoint;
     }
 
+    private BalBreakpoint toBalBreakpoint(SourceBreakpoint sourceBreakpoint, Source source) {
+        BalBreakpoint balBreakpoint = new BalBreakpoint();
+        balBreakpoint.setLine(sourceBreakpoint.getLine());
+        balBreakpoint.setSource(source);
+        balBreakpoint.setVerified(true);
+        balBreakpoint.setCondition(sourceBreakpoint.getCondition());
+        return balBreakpoint;
+    }
```

Comment:
Why cant we use the `SourceBreakpoint` instances itself instead of creating a custom breakpoint type, as it seems like `BalBreakpoint` is used only to hold a subset of information of the `SourceBreakpoint`?

Reply:
Removed custom `balBreakpoint` class and used `SourceBreakpoint` information